test/Waives.Pipelines.Tests/PipelineFacts.cs

```diff
@@ -254,5 +273,15 @@ await _sut.WithDocumentsFrom(source)
             Assert.Equal(1, onDocumentErrorCalledFor.First().sequence);
             Assert.Equal(2, onDocumentErrorCalledFor.Last().sequence);
         }
+
+        private static void AssertStreamsAreEqual(Stream expected, Stream actual)
+        {
+            int expectedByte, actualByte;
```

Comment:
would 
```csharp
var expectedBytes = await expected.ReadAsByteArrayAsync();
var actualBytes = await actual.ReadAsByteArrayAsync();
Assert.Equal(expectedBytes, actualBytes);
```
be any simpler, do you think?

Reply:
It would if there was a `ReadAsByteArrayAsync()` method! 😁 But it's made cleaner by comparing the bytes of a `MemoryStream`. Resolved in 894c70b.